whois-commons/src/main/resources/internals_schema.sql

```diff
@@ -71,6 +71,28 @@ CREATE TABLE `default_maintainer_sync_history` (
     `is_synchronised` tinyint(1) DEFAULT 0,PRIMARY KEY (`id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 
+
+DROP TABLE IF EXISTS `default_maintainer`;
```

Comment:
Is this the "in progress " table? Can it be named `default_maintainer_in_progress` to be explicit?